### ISSUE TITLE
Update spec definitions and capture+propagate in clinvar-combiner

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/clinvar-streams#15]
@theferrit32 commented on [Thu Sep 17 2020](https://github.com/clingen-data-model/clinvar-raw-producer/issues/14)

`rcv_accession.date_last_evaluated` is currently spec'ed as required, but is often missing from the record

---

@theferrit32 commented on [Thu Sep 17 2020](https://github.com/clingen-data-model/clinvar-raw-producer/issues/14#issuecomment-694425079)

google cloud filter for current pod example

```
resource.type="k8s_container"
resource.labels.project_id="clingen-dx"
resource.labels.location="us-east1-b"
resource.labels.cluster_name="prod-cluster"
resource.labels.namespace_name="default"
labels.k8s-pod/app="clinvar-raw-producer"
resource.labels.pod_name="clinvar-raw-producer-5f5db77689-xnw92"
textPayload=~"WARN.*:entity_type \"rcv_accession\""
```

---

@theferrit32 commented on [Wed Sep 23 2020](https://github.com/clingen-data-model/clinvar-raw-producer/issues/14#issuecomment-697439165)

not all `clinical_assertion` records have a `rcv_accession_id`

**Comment by larrybabb** [comment]
@theferrit32 do i understand this to be related to the need to fix the missing `rcv_accession_ids` on scvs? If so I will have this be blocked by a new ticket to fix the `trait_set` mappings that are missing which are the root cause of the missing fks for the `rcv_accession_ids` in the scv records.

Please help clarify my understanding of where this stands.

**Comment by theferrit32** [comment]
@larrybabb this is just about updating the clojure message specs based on things we've observed, and passing those messages down to genegraph. The `rcv<->scv` missing link described above might actually be something we want to keep in the spec as required, since it is actually a bug that it is sometimes missing. There might be other spec fields that need to be updated though.
